examples/yandex.js

```diff
@@ -0,0 +1,33 @@
+const fastify = require('fastify')({ logger: { level: 'trace' } })
```

Comment:
@fastify/plugins I think the examples/ folder is becoming too big to ship

I would switch to a single generic example and add a custom one only if necessary

Not necessary to do it in this PR tho 👍🏼